### DISCUSSION TITLE
[A/L] (2021/22) Funzioni - Esercizio 14

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#254] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152315639-0be9984f-5a70-4434-afce-1ca1db8451ca.png)

**Comment by LuginiAndrea** [comment]
Se $f$ non è iniettiva non possiamo dire nulla.
Se $f$ è iniettiva possiamo vedere che, al pari dell'intersezione, l'ipotesi è vera.
Preso un qualunque $y \in f\left(X\right)$, se questi è presente in $f\left(X\right) - f\left(A\right)$ vuol dire $y \notin f\left(A\right)$, e quindi che per $x \textrm{ t.c. } y = f\left(x\right)$ possiamo dire che $x \in X, x \notin A$. Sappiamo inoltre che, essendo $f$ iniettiva, le seguenti proposizioni sono vere
$$x \in A \Longleftrightarrow  f\left(x\right) \in f\left(A\right)$$
$$x \notin A \Longleftrightarrow f\left(x\right) \notin f\left(A\right)$$
Di conseguenza, se $y \notin f\left(X\right) - f\left(A\right)$, e quindi $y \in f\left(X\right), y \in f\left(A\right)$ ( non consideriamo gli $y \notin f\left(X\right)$ ), $x \notin X - A$.
Possiamo quindi riassumere il tutto come 
$$y \in f\left(X\right) - f\left(A\right) \Longleftrightarrow x \in X - A$$
$$y \notin f\left(X\right) - f\left(A\right) \Longleftrightarrow x \notin X - A$$
Possiamo quindi vedere la simmetria fra la differenza insiemistica delle immagini e delle proprie controimmagini.
Ergo l'ipotesi è vera
